api/Dockerfile

```diff
@@ -0,0 +1,91 @@
+
+FROM node:lts-buster-slim AS web
+
+ENV APP_ROOT /app_root
+
+WORKDIR ${APP_ROOT}
+COPY --chown=${USER} ./web .
+
+RUN npm ci 
+RUN npm test --  --runInBand --coverage --watchAll=false
+RUN npm run build 
+    
+FROM registry.access.redhat.com/ubi8/ubi:8.5-200 AS build
+
+ENV APP_ROOT /app_root
+ENV HOME /home
+
+WORKDIR ${APP_ROOT}
+USER root
+
+ENV MAVEN_VERSION 3.6.3
```

Comment:
suggestion: old version of maven. Is it possible to bump to 3.8.X ?

Reply:
Done